### DISCUSSION TITLE
Variable size list items

### DISCUSSION THREAD

**Discussion by LizardEatsFlies** [clauderic/dnd-kit#150] [Q&A] [answered]
So I have a vertical list of cards of various sizes and so far this library is awesome and its working beautifully using the sortable context, HOWEVER, im running into an issue where, when i drag an item over an item with a different height, the dragged item gets squished or stretched to the size of the item its being dragged over.

I have no idea whats causing this. Can anyone help?

Thanks!

![image](https://user-images.githubusercontent.com/18158232/112207828-d783d180-8bd4-11eb-8766-6515c0c4c79d.png)

**Comment by clauderic** [comment, marked as answer]
Hey @LizardEatsFlies, this is a common pitfall that I need to make sure to document better.

By default, `@dnd-kit` calculates the scale difference between the draggable item and the droppable it is currently over and adds it to the `scaleX` and `scaleY` properties of the `transform` that is returned in `useSortable` and `useDraggable`.

The behaviour you're seeing happens when using `CSS.Transform.toString(transform)` rather than `CSS.Translate.toString(transform)`. 

Most consumers will likely want to use `CSS.Translate.toString(transform)`, so I'll update documentation accordingly soon.

**Reply by LizardEatsFlies** [reply]
This is brilliant! Thank you so much! you are a life saver.

Here is a picture of my cute cat as gratitude 

![image](https://user-images.githubusercontent.com/18158232/112209469-b7551200-8bd6-11eb-9f31-fc4d5a35d062.png)

**Comment by darren10201** [comment]
did this fix it? @LizardEatsFlies
